src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:23454/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:23454/project/issues?id=sample&branch=feature2&issues=AWCrwJ7OwC2WnD8jumt-&open=AWCrwJ7OwC2WnD8jumt-)

[ISSUE_KEY]: AWCrwJ7OwC2WnD8jumt-